core/src/main/java/com/arcrobotics/ftclib/hardware/motors/Motor.java

```diff
@@ -238,6 +238,32 @@ public DcMotor.ZeroPowerBehavior getBehavior() {
     public Motor() {
```

Comment:
Just out of curiosity, why is there an empty default constructor.

Reply:
For OOP shenanigans